### PR TITLE
INDY-1856: Req handler restructuring

### PR DESCRIPTION
**Pull request by ArtObr**
Signed-off-by: ArtObr <artemobruchnikov@gmail.com>

### CODE REVIEW

**Comment by ArtObr** [comment]
This PR must be tested when https://github.com/hyperledger/indy-plenum/pull/1025 will be merged

File: indy_node/test/anon_creds/test_revoc_reg_entry_validation.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def test_validation_with_unexpected_accum(
         create_node_and_not_start):
     node = create_node_and_not_start
     req_entry = build_txn_for_revoc_def_entry_by_default
-    req_handler = node.getDomainReqHandler()
+    req_handler = node.init_domain_req_handler()
```
Comment (ashcherbakov) [inline indy_node/test/anon_creds/test_revoc_reg_entry_validation.py]: Why do we re-init it here? Should we just get already existent req handler?

**Comment by ArtObr** [comment]
test this please

File: indy_node/server/query_req_handler.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from plenum.server.req_handler import RequestHandler
+
+
+class QueryReqHandler(RequestHandler):
```
Comment (ashcherbakov) [inline indy_node/server/query_req_handler.py]: Why do we need it?

File: indy_node/server/req_handlers/query_req_handlers/get_attribute_handler.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from indy_node.server.query_req_handler import QueryReqHandler
+
+
+class GetAttributeHandler(QueryReqHandler):
```
Comment (ashcherbakov) [inline indy_node/server/req_handlers/query_req_handlers/get_attribute_handler.py]: It should extend ReadRequestHandler from Plenum, shouldn't it?

Comment (ArtObr) [reply indy_node/server/req_handlers/query_req_handlers/get_attribute_handler.py]: yes

**Comment by ArtObr** [comment]
test this please

**Pull request merged**